app/assets/javascripts/app/views/header_view.js

```diff
@@ -6,42 +6,21 @@ app.views.Header = app.views.Base.extend({
 
   className: "dark-header",
 
-  events: {
-    "click ul.dropdown li:first-child": "toggleUserDropdown",
+  events:{
     "focusin #q": "toggleSearchActive",
     "focusout #q": "toggleSearchActive"
   },
 
-  initialize: function(){
-    $(document.body).click($.proxy(this.hideUserDropdown, this));
-
+  initialize: function() {
     return this;
   },
 
   postRenderTemplate: function(){
-    new app.views.Notifications({ el: '#notification_dropdown' });
-    new app.views.NotificationDropdown({ el: '#notification_badge' });
+    new app.views.Notifications({ el: '#notifications-dropdown' });
+    new app.views.NotificationDropdown({ el: '#notification-badge' });
```

Comment:
Mixed double and single quotes.<br>Do not use 'new' for side effects.<br>'app' is not defined.